roles/georchestra/tasks/mviewerstudio.yml

```diff
@@ -1,7 +1,85 @@
 ---
-- name: Mviewerstudio accessible only if connected
+- name: Install mviewerstudio dependencies
+  apt:
+    pkg: 
+      - git
+      - libxslt1-dev
+      - libxml2-dev
+      - python3
+      - python3-pip
+      - python3-venv
+    state: present
+
+- name: Create a group georchestra
+  become: yes
+  group: 
+    name: georchestra
+    state: present
+
+- name: Create the user for mviewer and mviewerstudio
+  become: yes
+  user:
+    name: mviewer
+    state: present
+
+- name: Checkout mviewerstudio
+  git:
+    repo: "{{ mviewerstudio.gitrepo }}"
+    version: "{{ mviewerstudio.gitversion }}"
+    dest: /srv/apps/mviewerstudio/
+
+- name: Change ownership
+  file:
+    path: /srv/apps/mviewerstudio/
+    owner: mviewer
+    group: georchestra
+    state: directory
+    recurse: yes
```

Comment:
same as below, no need for owner/group and it shouldnt need write access to its code, only its separate data dir.